module/fork_spec.go

```diff
@@ -62,17 +71,21 @@ func GetForkParameters(network Network) []*ForkSpec {
 	switch network {
 	case Localnet:
 		hardForks[indexPascalHF].HeightOrTimestamp = &ForkSpec_Height{Height: 0}
-		hardForks[indexLorentzHF].HeightOrTimestamp = localLatestHF
+		hardForks[indexLorentzHF].HeightOrTimestamp = &ForkSpec_Height{Height: 1}
+		hardForks[indexMaxwellHF].HeightOrTimestamp = localLatestHF
 		return hardForks
 	case Testnet:
 		hardForks[indexPascalHF].HeightOrTimestamp = &ForkSpec_Height{Height: 48576786}
 		hardForks[indexLorentzHF].HeightOrTimestamp = &ForkSpec_Height{Height: 49791365}
+		// https: //github.com/bnb-chain/bsc/blob/256d8811f441c29cb0812943dc660ac13192829c/params/config.go#L239
```

Comment:
[nits] unexpected space

```suggestion
		// https://github.com/bnb-chain/bsc/blob/256d8811f441c29cb0812943dc660ac13192829c/params/config.go#L239
```

Reply:
Fixed by https://github.com/datachainlab/ibc-parlia-relay/commit/9cd6c5516f9c8625f40c2007d4bedde7b32f76e6